src/badguy/crusher.cpp

```diff
@@ -123,31 +123,40 @@ HitResponse
 Crusher::collision(GameObject& other, const CollisionHit& hit)
 {
   auto* player = dynamic_cast<Player*>(&other);
+  bool crushed_bottom = m_state == CRUSHING && !m_sideways && hit.bottom;
+  bool crushed_sideways = m_state == CRUSHING && m_sideways &&
+    ((hit.left && m_physic.get_velocity_x() < 0.f) ||
+     (hit.right && m_physic.get_velocity_x() > 0.f));
+  bool is_crushing = crushed_bottom || crushed_sideways;
 
   // If the other object is the player, and the collision is at the
   // bottom of the crusher, hurt the player.
```

Comment:
This comment doesn't reflect the changes (sideways collision also now hurts the player).